test.py

```diff
@@ -10,53 +10,7 @@
 import time
 import warnings
 
-
-def func_exists(mod_path, func_name):
-    try:
-        with open(mod_path, "r") as file:
-            module_content = file.read()
-    except FileNotFoundError:
-        return False  # Module file not found
-
-    try:
-        tree = ast.parse(module_content)
-    except SyntaxError:
-        return False  # Syntax error in module
-
-    for node in ast.walk(tree):
-        if isinstance(node, ast.FunctionDef) and node.name == func_name:
-            return True
-    return False
-
-
-def import_sdp_mods(include=None, ignore=None):
-    mods = []
-    for m in sorted(list(pkgutil.iter_modules(sdp.__path__))):
-        mod_path = f"sdp/{m[1]}.py"
-        if (
-            include is not None
-            and len(include)
-            and not any(mod in mod_path for mod in include)
-        ):
-            continue
-        if (
-            ignore is not None
-            and len(ignore)
-            and any(mod in mod_path for mod in ignore)
-        ):
-            continue
-
-        if (
-            "sdp" in m[1]
-            and func_exists(mod_path, "sliding_dot_product")
-            and func_exists(mod_path, "setup")
-        ):
-            mod_name = f"sdp.{m[1]}"
-            mod = importlib.import_module(mod_name)
-            mods.append(mod)
-
-    return mods
-
+from utils import import_sdp_mods
```

Comment:
1. Let's just `import utils` as writing `import import_sdp_mods` looks bad
2. Now that we have unit tests, let's rename this `test.py` file to `timing.py` (and reserve "test" related things for unit testing)

Reply:
Yeah... was thinking about `test_performance.py` but wanted to avoid `test `... `timing.py` is a good name! 👍